gcovr/configuration.py

```diff
@@ -915,6 +915,16 @@ def merge_options_and_set_defaults(
         default=None,
         const=OutputOrDefault(None),
     ),
+    GcovrConfigOption(
+        "template_dir",
+        ["--template-dir"],
```

Comment:
Use a prefix since this is hor HTML only.
```suggestion
        "html_template_dir",
        ["--html_template-dir"],
```

Reply:
@Spacetown did you mean `--html-template-dir` here ? Underscore seems weird.

Reply:
The option always uses `-` where the key uses `_`.